Cargo.toml

```diff
@@ -10,4 +10,5 @@ members = [
     "infrastructure/comms",
     "infrastructure/crypto",
     "infrastructure/storage",
+    "MerkleMountainRange"
```

Comment:
Casing is wrong here. Also, does this not fall under one of the existing folders